tests/unit/bluetooth/at/prj.conf

```diff
@@ -3,3 +3,4 @@ CONFIG_BLUETOOTH_BREDR=y
 CONFIG_BLUETOOTH_HFP_HF=y
 CONFIG_NET_BUF=y
 CONFIG_ZTEST=y
+CONFIG_SERIAL=y
```

Comment:
This usually comes from the board, probably it would have some side effects on boards without serial, @jhedberg any input?